app/views/actors/_form.html.erb

```diff
@@ -0,0 +1,22 @@
+<%= form_for(actor) do |f| %>
```

Comment:
I am getting an error "Production must exist", which means that there should be a select field for 'Production' (movie) in order for the form to validate when being created.